### PR TITLE
[SW-2008] Fix compile after removing code for external backend from H2O

### PR DESCRIPTION
**Pull request by jakubhava**
We were still referencing one class from H2O

### CODE REVIEW

File: extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/Utils.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling.extensions.serde
+
+import water.fvec.Vec
+
+object Utils extends ChunkSerdeConstants {
```
Comment (mn-mikke) [inline extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/Utils.scala]: note: `Utils` sounds a bit generic. I'm wondering whether we shouldn't go for anything more specific to serialization/deserialization although the class is in the `ai.h2o.sparkling.extensions.serde` namespace. 

This note is  really really minor and you can leave it as it's.

Comment (jakubhava) [reply extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/Utils.scala]: Good point, what about `SerdeUtils`?

Comment (mn-mikke) [reply extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/Utils.scala]: Sounds good to me ;-)

File: extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/SerdeUtils.scala [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling.extensions.serde
+
+import water.fvec.Vec
+
+object SerdeUtils extends ChunkSerdeConstants {
```
Comment (mn-mikke) [inline extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/SerdeUtils.scala]: I also wondering what is better? inheritance or `import ai.h2o.sparkling.extensions.serde. ChunkSerdeConstants._` WDYT?

Comment (jakubhava) [reply extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/SerdeUtils.scala]: I like `import ChunkSerdeConstants._` also, good catch

**Pull request merged**